wordpress-mobile/WordPress-iOS
# Issue/7601 media video settings

Fixes #7601 

To test:
 - Go to Me ->App Settings
 - Change the Video Size settings
 - Check if going in and out the setting is correctly saved
 - Go to Blog -> Posts
 - Create a new Post
 - Add a video to the post
 - Check that the video export process goes well
 - Check in the server if the video respected the maximum upload video size
 - Remember the settings doesn't upscale videos!
